pf9/cluster/commands.py

```diff
@@ -474,18 +510,26 @@ def prepnode(ctx, user, password, ssh_key, ips, floating_ip):
     """
     logger.info(msg=click.get_current_context().info_name)
 
-    # Load active config
-    Get(ctx).active_config()
-    # Validate Token
-    Get(ctx).get_token_project_user_id()
-
     segment_session = SegmentSession()
     segment_event_properties = dict(arg_user='REDACTED', arg_password='REDACTED', arg_ssh_key=ssh_key,
-                                    arg_ips=ips, arg_floating_ip=floating_ip, keystone_user_id=ctx.params['user_id'],
-                                    du_account_url=ctx.params['du_url'])
+                                    arg_ips=ips, arg_floating_ip=floating_ip)
     SegmentSessionWrapper(ctx).load_segment_session(segment_session, segment_event_properties, "Prep Node")
+
+    try:
+        # Load active config
+        Get(ctx).active_config()
+        # Validate Token
+        Get(ctx).get_token_project_user_id()
+    except Exception as e:
+        click.secho("Encountered an error while preparing the provided nodes as "
+                    "Kubernetes nodes. {}".format(e), fg="red")
+        SegmentSessionWrapper(ctx).send_track_error('Load Active config', e)
+        sys.exit(1)
+
+    SegmentSessionWrapper(ctx).reload_segment_session_with_auth()
     SegmentSessionWrapper(ctx).send_track('Load Active config')
     segment_session.send_identify(ctx.params['du_username'], ctx.params['user_id'])
+    segment_session.send_group(ctx.params['user_id'], ctx.params['du_url'])
```

Comment:
No need to track L539?

Reply:
Probably not as, it is a user's decision to not go further in the operation